include/logging/log_output.h

```diff
@@ -50,6 +50,15 @@ extern "C" {
  */
 #define LOG_OUTPUT_FLAG_FORMAT_SYST		BIT(7)
```

Comment:
shouldn't this flag be deprecated?

Reply:
I think this flag is still being used in parts of v1 logging routines (put, sync_string, sync_hexdump) in the backends.

Reply:
ok, can you add comment  that it is deprecated and can be used only when `CONFIG_LOG1` is enabled. It will be easier to find it when v1 will be removed after 3.1.

Reply:
Done.

Reply:
@MaureenHelm Using __DEPRECATED_MACRO for LOG_OUTPUT_FLAG_FORMAT_SYST results in warnings which cause failing testcases in twister run.